demo/2025-10/README.md

```diff
@@ -1,227 +1,78 @@
-# Prototype demo - October 2025
+# Run the demo
 
-Minimum viable demo of Leios network traffic interfering with Praos using a three node setup and prepared Praos and Leios data.
```

Comment:
I thought having a quick intro sentence of what this is can't hurt?

Also I would like to add the overview diagram here.

Reply:
Added it now @bladyjoker https://github.com/input-output-hk/ouroboros-leios/commit/6c0972f6fb629dab22be2e70ee37e1122e3ea88f